app/src/main/java/org/mozilla/rocket/urlinput/SearchPortalViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package org.mozilla.rocket.urlinput
+
+import android.arch.lifecycle.MediatorLiveData
+import android.arch.lifecycle.ViewModel
+
+class SearchPortalViewModel(repository: SearchPortalRepository) : ViewModel() {
+
+    val searchPortalObservable = MediatorLiveData<ArrayList<SearchPortal>>()
+    private val liveGlobal = repository.fetchGlobal()
```

Comment:
Remind me to see if we can use Transformation here. (instead of manually merge portals